client/my-sites/email/email-management/index.jsx

```diff
@@ -130,15 +126,9 @@ class EmailManagement extends React.Component {
 	}
 
 	content() {
-		const {
-			domains,
-			hasGSuiteUsersLoaded,
-			hasSiteDomainsLoaded,
-			isFetchingSiteDomains,
-			selectedDomainName,
-		} = this.props;
-
-		if ( ! hasGSuiteUsersLoaded || ! hasSiteDomainsLoaded || isFetchingSiteDomains ) {
+		const { domains, hasGSuiteUsersLoaded, hasSiteDomainsLoaded, selectedDomainName } = this.props;
+
+		if ( ! hasGSuiteUsersLoaded || ! hasSiteDomainsLoaded ) {
```

Comment:
I've updated that code to avoid displaying the placeholder when site domains are fetching. This solves an issue when we reload the `Email` page where the placeholder would be displayed twice.